extensions/wechat/src/types/images.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.png" {
+  const value: string;
+  export default value;
+}
```

Comment:
Could you explain what the file was used for?

I don't think Raycast needs it.

Reply:
Due to a bug in WeChatExtension itself, some friends' avatars cannot be obtained, so a default avatar is added.

Reply:
This not affect the code,I've removed it.